### PR TITLE
Modified based on combo runner fix

### PR DESCRIPTION
**Pull request by zapion**
- Adapting combo runner fix.
- Supporting parameter from both envrion and settings.json

### CODE REVIEW

**Comment by zapion** [comment]
@ShakoHo , @shinglyu r?

File: mtbf_job_runner.py [hunk 1]
```diff
@@ -87,11 +87,16 @@ def add_7mobile_action(self):
...
     @action(enabled=False)
     def change_memory(self):
-        #TODO: use native adb/fastboot command to change memory?
+        # This function only work in flame
+        # TODO: use native adb/fastboot command to change memory?
         # Make sure it's in fastboot mode, TODO: leverage all fastboot command in one task function
+        memory = 512 # default set 512
+        if 'MEM' in os.environ:
+            memory = os.environ['MEM']
+        elif self.settings['change_memory']['enabled'] and 'memory' in self.settings['change_memory']:
```
Comment (shinglyu) [inline mtbf_job_runner.py]: So if environ is set, the settings.json setting will be ignored? Is that what you intended?

Comment (shinglyu) [inline mtbf_job_runner.py]: I just want to clarify that. Otherwise it's an r+ :)

**Comment by ShakoHo** [comment]
verified in my local environment. r+

File: mtbf_job_runner.py (hunk 1 above)
Comment (zapion) [inline mtbf_job_runner.py]: good point and encourage you to provide question while reviewing.

I suppose the priority is more dynamic > static, for instance

command line argument > file loaded argument > system default

if you're an operation engineer, when you find something wrong you might need to find correct combination of arguments and then modify them in your config files.

**Pull request merged**